io/src/main/scala/fs2/io/JavaInputOutputStream.scala

```diff
@@ -144,9 +144,9 @@ private[io] object JavaInputOutputStream {
 
       def go(acc:Array[Byte]):F[Int] = {
         F.flatMap(readOnce(acc,0,1,queue,dnState)) { read =>
-          if (read < 0) F.pure(read)
```

Comment:
The `InputStream` documentation explicitly says that `-1` should be returned if there is no more data.
It would also be possible to do ` if (read < 0) F.pure(-1)` but i thought it would be clearer this way since  `readOnce` also explicitly uses `-1` so i would not expect it to return any other negative number here.